web/styles/compose.css

```diff
@@ -889,6 +889,23 @@ input.recipient_box {
         margin-right: 7px;
         margin-left: 7px;
     }
+
+    .compose_buttons_container {
+        height: 28px;
```

Comment:
Again, let's either make variables or document the origin of dimensions like this.